.github/workflows/build-jax-wheels.yml

```diff
@@ -0,0 +1,86 @@
+name: Build Jax Wheels
+
+on:
+  push:
+    branches: [main]
+    tags: ["*"]
+  pull_request:
+    # Check all PR
+
+#env:
+#  SPHERICART_NO_LOCAL_DEPS: "1"
+
+jobs:
+  build-jax-wheels:
+    runs-on: ubuntu-latest
```

Comment:
could you give the job a name with something like `name: ${{matrix.os}} ${{matrix.arch}} Python  ${{matrix.python-version}} Torch ${{matrix.torch-version}} ...` (here and in the torch wheels build)

Reply:
This makes it a lot easier to read when dealing with a very large list of jobs!

Reply:
added this in!